includes/modules/pages/index/header_php.php

```diff
@@ -37,41 +38,41 @@
             $current_category_is_disabled = true;
         }
     }
-    $category_products_query =
+    $category_products_query = 
         "SELECT products_id
-         FROM " . TABLE_PRODUCTS_TO_CATEGORIES . "
-         WHERE categories_id = :currentCategoryId
-         LIMIT 1";
+           FROM " . TABLE_PRODUCTS_TO_CATEGORIES . "
+          WHERE categories_id = :currentCategoryId
+          LIMIT 1";
     $category_products_query = $db->bindVars($category_products_query, ':currentCategoryId', $current_category_id, 'integer');
     $category_products = $db->Execute($category_products_query);
     if (!$category_products->EOF) {
         $current_category_has_products = true;
     } else {
         $category_parent_query =
             "SELECT parent_id
-             FROM " . TABLE_CATEGORIES . "
-             WHERE parent_id = :currentCategoryId
-             LIMIT 1";
+               FROM " . TABLE_CATEGORIES . "
+              WHERE parent_id = :currentCategoryId
+              LIMIT 1";
         $category_parent_query = $db->bindVars($category_parent_query, ':currentCategoryId', $current_category_id, 'integer');
         $category_parent = $db->Execute($category_parent_query);
         $current_category_has_subcats = !$category_parent->EOF;
     }
-
+    
     // -----
     // Give an observer the chance to override the default handling for the category.
     //
     $category_redirect_handled = false;
     $zco_notifier->notify(
-        'NOTIFY_INDEX_CATEGORY_STATUS_CHECK',
-        ['cPath' => $cPath, 'current_category_id' => $current_category_id],
+        'NOTIFY_INDEX_CATEGORY_STATUS_CHECK', 
+        array('cPath' => $cPath, 'current_category_id' => $current_category_id),
```

Comment:
Why has this array convention been undone here, but continued to be used in the new code?

If modifying this, would say that the last array element should have a comma that follows.

Actually, don't see why this file is included in this commit at all (meaning nothing is really changed here that applies to what is being done, looks like this was accidentally included in the commit).

Reply:
this is a mistake in my git handling.  again, apologies.  this page should not have been modified at all by this branch.